module changed on disk: /home/lemaigna/applis/blender-2.74-rc2-linux-glibc211-x86_64/2.74/scripts/addons/object_print3d_utils/__init__.py reloading...


================================

Pre-assessment for group Z


================================

Global unit scale: 0.0010000000474974513
	 Fine, millimeters


--------------------------------



--------------------------------

Object <Tree>:
	Raw dimensions (mm): 39, 39, 99.00254821777344
	Volume: 157729mm3
	Object use texture <Texture>
	3D toolbox analyse:
		Non Manifold Edge: 0
		Bad Contig. Edges: 0
[WW]		Intersect Face: 21
[WW]		Zero Faces: 54
[WW]		Zero Edges: 1
[WW]		Non-Flat Faces: 448
[WW]		Thin Faces: 22
[WW]		Sharp Edge: 4
[WW]		Overhang Face: 27
	Exporting to <Tree.stl>...
		Export ok


================================

SUMMARY

- 1 part(s) submitted
- 1 part(s) use a texture
- total volume: 157729
	 Ok, below limit
- 7 warnings
- 0 errors
found bundled python: /home/lemaigna/applis/blender-2.74-rc2-linux-glibc211-x86_64/2.74/python
read blend: /home/lemaigna/src/cs211-visual-computing/milestone 2/pre-assessment/../submissions/Z//Finished tree rond 2.blend

Blender quit
